src/resources/SchemaService/SchemaServiceInterfaces.ts

```diff
@@ -1,27 +1,34 @@
 import {SourceModel, CreateSourceModel, CreateSourceOptions} from '../Sources/SourcesInterfaces';
 
-export interface SchemaEntities {
-    entities: SimpleSchemaEntity[];
+export interface SimpleSchemaEntity {
+    id: string;
+    displayName: string;
 }
```

Comment:
Tu devrais avoir le `recordCount` ici je crois! Right @agagnon2 ?

```suggestion
    displayName: string;
    recordCount: int;
}
```

Reply:
@mikegron je pense pas, selon [la doc](https://coveord.atlassian.net/wiki/spaces/WORK/pages/2052719149/Schema+Service+documentation#API-Documentation) ce sera dans le `/entities/:entityId`. Je vais le mettre en field optionnel comme ça on pourra utiliser la même interface et ça évitera de se retrouver avec encore plus de types

Reply:
Parfait!